tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Steps/ServiceSearchSteps.cs

```diff
@@ -16,64 +16,86 @@ namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Steps;
 /// </summary>
 public class ServiceSearchSteps
 {
-    readonly ConfigModel config;    
-    private readonly string baseUrl;
-    private ServiceSearchRequest request;
-    private HttpResponseMessage lastResponse;
-    private HttpStatusCode statusCode;
-    private const string serviceSearchPath = "api/metrics/service-search";
-    public ServiceSearchSteps()
-    {
-        config = ConfigAccessor.GetApplicationConfiguration();
-        baseUrl = config.BaseUrl;
-    }
+  readonly ConfigModel config;  
+  private readonly string baseUrl;
```

Comment:
refactor: We should be following the naming conventions with all these variables, all private members should be prefixed with a `_` e.g., `_baseUrl`, `_request`, etc. Const variables are PascalCase, e.g., `ServiceSearchPath`